frappe/patches/v10_0/modify_naming_series_table.py

```diff
@@ -0,0 +1,16 @@
+
+'''
+    Modify the Integer 10 Digits Value to BigInt 20 Digit value
+    to generate long Naming Series
+
+'''
+import frappe
+import re
+def execute():
+        
+        fields = frappe.db.sql("""DESC `tabSeries` """, as_dict=True)
+        for field in fields:
```

Comment:
why this loop?

Reply:
@rmehta 

1) It's just two columns table now(Name(varchar), Current(int)), May be in future if users want to have more than one Integer column in same series table then It may alter the other Integer fields too, to prevent this issue I have added loop over two columns to compare the fieldname and datatype.
2.) It'll compare current field datatype to ensure it whether user has already changed it bigint or higher datatype or not. If I'ts already change then no alteration is required.
I have tested patch and it's working fine.
## Results
```
MariaDB [d502ac186a03513d]> alter table `tabSeries` modify current bigint;
Query OK, 31 rows affected (0.99 sec)              
Records: 31  Duplicates: 0  Warnings: 0

MariaDB [d502ac186a03513d]> alter table `tabSeries` modify current bigint;
Query OK, 0 rows affected (0.00 sec)
Records: 0  Duplicates: 0  Warnings: 0
```